src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadThreadingAssemblerArgumentCollection.java

```diff
@@ -23,7 +24,11 @@
      */
     @Advanced
     @Argument(fullName="kmerSize", shortName="kmerSize", doc="Kmer size to use in the read threading assembler", optional = true)
-    public List<Integer> kmerSizes = Arrays.asList(10, 25);
+    public List<Integer> kmerSizes = new ArrayList<>();
```

Comment:
are these changes needed because of serialization? If yes, does this mean you're pulling in the argument to serialization via a closure (closures are always serialized by Java serialization and not Kryo)? Is yes, maybe that's a problem - we should not avoid serializing big closures